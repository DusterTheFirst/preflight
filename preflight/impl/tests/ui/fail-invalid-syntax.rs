use preflight_impl::{avionics_harness, Avionics, Control, Sensors};

struct Controller;

#[avionics_harness(ooooo , eee / 0099 | gee)]
impl Avionics for Controller {
    fn guide(&mut self, _: &Sensors) -> Option<Control> {
        todo!()
    }
}

fn main() {}
