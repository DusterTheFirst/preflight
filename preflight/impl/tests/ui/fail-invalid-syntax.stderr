error: expected `,`
 --> $DIR/fail-invalid-syntax.rs:5:32
  |
5 | #[avionics_harness(ooooo , eee / 0099 | gee)]
  |                                ^

warning: unused imports: `Avionics`, `Control`, `Sensors`
 --> $DIR/fail-invalid-syntax.rs:1:40
  |
1 | use preflight_impl::{avionics_harness, Avionics, Control, Sensors};
  |                                        ^^^^^^^^  ^^^^^^^  ^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
